src/assets/scss/Profile.scss

```diff
@@ -0,0 +1,91 @@
+$circular: 50%;
+$square-image: 220px;
+$btn-display: inline-block;
+$btn-font-weight: 400;
+$btn-margin-left: 5px;
+$text-align: center;
+$vertical-align: middle;
+$btn-webkit-user-select: none;
+$btn-moz-user-select: none;
+$btn-ms-user-select: none;
+$btn-user-select: none;
+$btn-padding: .375rem .75rem;
+$btn-font-size: 1rem;
+$btn-line-height: 1.5;
+$btn-border-radius: .25rem;
+$red-color: red;
+$btn-font-size: 1rem;
+$btn-line-height: 1.5;
+$btn-border-radius: .25rem;
+$white-color: #fff;
+$blue-color: #023873;
+
+
+.view-profile-form {
+  min-height: 100%;
+  margin: 120px auto 100px;
+}
+
+@media screen and (min-width: 1024px) {
+  .view-profile-form {
+    min-height: 100%;
+    margin: 120px auto 470px;
+  }
```

Comment:
This margin is too big and it is causing too much spacing between the form and footer
![image](https://user-images.githubusercontent.com/22131999/61707623-98dcaa80-ad53-11e9-8f9a-261d42e05b3d.png)